### ISSUE TITLE
Labels on BoundingBoxes3D

### ISSUE THREAD

**Issue by Ardorax** [Visual-Behavior/aloception-oss#224]
BoundingBoxes3D doesn't have a 'append_labels' method like BoundingBoxes2D.
```py
import numpy as np
import torch
from aloscene import Frame, Labels, BoundingBoxes3D, CameraExtrinsic, CameraIntrinsic

frame = Frame(np.zeros((3, 500, 500)), normalization="01")
frame.append_cam_intrinsic(CameraIntrinsic(focal_length=721.0, plane_size=frame.HW))
frame.append_cam_extrinsic(CameraExtrinsic(torch.eye(4, dtype=torch.float32)))
label = Labels([1], names=["name"])

# Labels can be added to BoundinBoxes3D by using the `labels` argument
box3d = BoundingBoxes3D([[10, 10, 400, 80, 46, 18, 1]], labels=label)

# BoundinBoxes3D doesn't have a 'append_labels' method
# box3d.append_labels(label)

frame.append_boxes3d(box3d)

frame.get_view().render()
```
